contracts/LiquidityMining/README.md

```diff
@@ -36,11 +36,11 @@
 - On Ethereum, there is a `VotingController` contract to control the voting on incentives for the different markets on the different chains.
 - Adding a new chain:
   - Only the governance address will be able to add markets and a new chains' GaugeController.
-- There is a list of `(market)` that are elligible for the voting, added by governance
+- There is a list of `(market)` that are eligible, illegible for the voting, added by governance
```

Comment:
Correction of this sentence should be: "There is a list of (market) that are eligible or ineligible for voting, added by governance."
"illegible" means unreadable, but the correct term here is "ineligible."